doc/tutorials/pre-launched-rt.rst

```diff
@@ -30,60 +30,60 @@ Example of Pre-Launched RT
 mode must boot using GRUB.  The ACRN hypervisor is loaded as a GRUB
-multiboot kernel, while the Pre-Launched RT kernel and Service VM
-kernels are loaded as multiboot modules. The ACRN hypervisor, Service
-VM, and Pre-Launched RT kernel images are all located on the NVMe drive.
+multiboot kernel, while the pre-launched RTVM kernel and Service VM
+kernel are loaded as multiboot modules. The ACRN hypervisor, Service
+VM, and pre-launched RTVM kernel images are all located on the NVMe drive.
 We recommend installing Ubuntu on the NVMe drive as the Service VM OS,
 which also has the required GRUB image to launch Pre-Launched RT mode.
-Refer to :ref:`gsg`, to
-install Ubuntu on the NVMe drive, and use grub to launch the Service VM.
+Refer to :ref:`gsg` to
+install Ubuntu on the NVMe drive, and use GRUB to launch the Service VM.
 
 Install Pre-Launched RT Filesystem on SATA and Kernel Image on NVMe
 ===================================================================
 
-Follow the :ref:`gsg` to install RT rootfs on SATA drive.
+Follow the :ref:`gsg` to install RT rootfs on the SATA drive.
 
-The Kernel should
+The kernel should
 be on the NVMe drive along with GRUB. You'll need to copy the RT kernel
 to the NVMe drive. Once you have successfully installed and booted
 Ubuntu from the NVMe drive, you'll then need to copy the RT kernel from
 the SATA to the NVMe drive:
 
 .. code-block:: none
 
-   # mount /dev/nvme0n1p1 /boot
-   # mount /dev/sda1 /mnt
-   # cp /mnt/bzImage /boot/EFI/BOOT/bzImage_RT
+   sudo mount /dev/nvme0n1p1 /boot
+   sudo mount /dev/sda1 /mnt
+   sudo cp /mnt/bzImage /boot/EFI/BOOT/bzImage_RT
 
 Build ACRN With Pre-Launched RT Mode
 ====================================
 
 The ACRN VM configuration framework can easily configure resources for
-Pre-Launched VMs. On Whiskey Lake WHL-IPC-I5, to passthrough SATA and
-Ethernet 03:00.0 devices to the Pre-Launched RT VM, build ACRN with:
+pre-launched VMs. On Whiskey Lake WHL-IPC-I5, to pass through SATA and
+Ethernet 03:00.0 devices to the pre-launched RTVM, build ACRN with:
 
 .. code-block:: none
 
    make BOARD_FILE=$PWD/misc/acrn-config/xmls/board-xmls/whl-ipc-i5.xml SCENARIO_FILE=$PWD/misc/acrn-config/xmls/config-xmls/whl-ipc-i5/hybrid_rt.xml RELEASE=0
 
 After the build completes, update ACRN on NVMe. It is
-/boot/EFI/BOOT/acrn.bin, if /dev/nvme0n1p1 is mounted at /boot.
+``/boot/EFI/BOOT/acrn.bin``, if ``/dev/nvme0n1p1`` is mounted at ``/boot``.
 
 Add Pre-Launched RT Kernel Image to GRUB Config
 ===============================================
 
-The last step is to modify the GRUB configuration file to load the Pre-Launched
-kernel. (For more information about this, see
-the :ref:`gsg`.) The grub config file will look something
+The last step is to modify the GRUB configuration file to load the pre-launched
+RTVM kernel. (For more information about this, see
+the :ref:`gsg`.) The GRUB configuration file will look something
```

Comment:
Similar here, we do not have those details in the GSG anymore.

Reply:
We could just update (if needed) the Grub configuration file and remove the reference to the GSG. We need the engineering team input for the right content, we need two `module2` lines, one for each kernel, but I'm not entirely sure if we need the ACPI stuff (e.g. `module2 /boot/ACPI_VM0.bin ACPI_VM0`)

@NanlinXie , who can guide us here?

Reply:
@NanlinXie said: [Nanlin:] Geoffroy has suggestion on how to refine the guide for User VM and launch script, I suggest we fix the terminology issue in this release and refine the guide next release.
@gvancuts Do you agree with Nanlin's suggestion?

Reply:
For the Grub configuration file, let's insert it in this PR before it gets merged. All we need is to get the right content from the engineering team, no need to push this out by a release. This guide is broken without this.

Reply:
Update: @gvancuts will fix in a future release. We can merge now for 2.7.